### ISSUE TITLE
Option page reboots device

### ISSUE THREAD

**Issue by Designbase** [alf45tar/PedalinoMini#315]
Hi,

I am using a LilyGO TTGO T8 ESP32 connected with WROVER KIT.
It is working fine except as soon as I enter the Option page the device reboots.
Did a fresh install took disconnected Leds and Switches but it is still rebooting.
Installed via webbrowser...

Any idea where this problem might come from?
Thanks!

**Comment by alf45tar** [comment]
Which firmware version did you used? esp32dev or what else?

**Comment by Designbase** [comment]
ESP-WROVER-KIT 2.3.2 as shown in the Installer

**Comment by alf45tar** [comment]
Did you tried with ttgo-t-eight?

**Comment by Designbase** [comment]
No that option does not work it sets the device in a boot loop...

**Comment by alf45tar** [comment]
Your board is not directly supporter. Try esp32dev or modify the pins assignment in Pedalino.h.

**Comment by Designbase** [comment]
Well the esp32dev did work... Why I used the Wrover is beyond me. Thought I read it somewhere....
Thanks again for your help!

**Comment by Designbase** [comment]
Mmmmm... still very unstable, random reboots.
Will look into the pin modification... if I can find it...

**Comment by Designbase** [comment]
What Board would you advise that has A. Battery connection and B. External Antenna...
and is compatible with the PedalinoMini ?

For now the LilyGO TTGO T8 ESP32 works and is stable build with the Wrover setting, except for the Option page. 

Thanks

**Comment by mattzzw** [comment]
Having the same issue using the DOIT ESP32 board (`esp32doit-devkit-v1`). 
Reboot happens every time I try to access the options page. It does not happen directly after a factory reset.
I tried it with the provided firmware and with the one I compiled myself.
Log output shows that the watchdog got triggered:
```
E (195047) task_wdt: Task watchdog got triggered. The following tasks did not reset the watchdog in time:
E (195047) task_wdt:  - async_tcp (CPU 0/1)
E (195047) task_wdt: Tasks currently running:
E (195047) task_wdt: CPU 0: IDLE0
E (195047) task_wdt: CPU 1: IDLE1
E (195047) task_wdt: Aborting.
abort() was called at PC 0x401c3864 on core 0

ELF file SHA256: 0000000000000000

Backtrace: 0x400907a0:0x3ffbffe0 0x40090a1d:0x3ffc0000 0x401c3864:0x3ffc0020 0x4008ed29:0x3ffc0040 0x401fea27:0x3ffbc2c0 0x401c58bb:0x3ffbc2e0 0x400931ed:0x3ffbc300 0x40091a2e:0x3ffbc320
```

**Comment by mattzzw** [comment]
Hm, the watchdog timeout is not exactly reproducible in 2.3.2 or 2.4.0.
Sometimes(tm) it works. Dunno if it has to do with the number of wifi networks to scan or something odd like that. But in general it reboots like 90% of the time for me when I try to access the options tab.

**Comment by alf45tar** [comment]
How many wifi network are reacheable? Can you try in a different location where there are few wifi network?

**Comment by alf45tar** [comment]
Just released version 2.4.1. It is a try to fix Options menu reboot in case of too much wifi networks.
It works for me like the previous version because I cannot simulate the real scenario.
Can you @mattzzw test and confirm if it works for you too?

**Comment by alf45tar** [comment]
I did a small change. I removed the while loop. A manual refresh of Options page is requested when scanning is finished.

**Comment by mattzzw** [comment]
Hm, I erased the ESP, flashed 2.4.1 with your latest commit and it seemed to work fine at first. The options page was reliably accessible.

Then I played around with changing configurations, uploaded my configuration and saved it as current profile and the options page did not load again. Just a part of the menu bar was shown.

<img width="611" alt="options" src="https://user-images.githubusercontent.com/1312120/155898220-cd2e08ef-ac95-4eea-9ac5-cac0f36313aa.png">


(So that for me points away from the number of wifi networks.)
Log output shows 
```
[W][AsyncTCP.cpp:962] _poll(): ack timeout 4
```
